src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java

```diff
@@ -122,6 +122,14 @@ protected void init() {
         c.addFilter(mf, "/*", 1);
         c.setResourceBase((new File(ScoreBoardManager.getDefaultPath(), "html")).getPath());
 
+        sh = new ServletHolder("game-data", new DefaultServlet());
+        sh.setInitParameter("cacheControl", "no-cache");
+        sh.setInitParameter("etags", "true");
+        sh.setInitParameter("resourceBase", (new File(ScoreBoardManager.getDefaultPath(), "game-data")).getPath());
+        sh.setInitParameter("dirAllowed", "true");
+        sh.setInitParameter("pathInfoOnly", "true");
+        c.addServlet(sh, "/game-data/*");
```

Comment:
Rather than doing this, I'd suggest moving the game-data directory under html.

Reply:
In which case we could simply integrate it as a new type of Media, which would also address your concerns regarding refresh and iframe.

Reply:
I'm not suggesting it be a type of media, just a link like we do for /custom/